src/core/drive/visit.ts

```diff
@@ -251,6 +261,15 @@ export class Visit implements FetchRequestDelegate {
     }
   }
 
+  goToSamePageAnchor() {
```

Comment:
I wonder if there's a better name for this? It originates from [the original code](https://github.com/turbolinks/turbolinks/pull/285/files#diff-d34bf5280590ab473d1ad488821d664a0746c3459a2d1f3c2d7ea6037be2b042R169-R177) that actually did the scrolling, but now this is handled by `render`.

Reply:
I think that's still an accurate naming as even if the actual jump is done by render, it's triggered here.